www/secretary/workbench/templates/emeritus-request.erb

```diff
@@ -0,0 +1,9 @@
+Dear <%= @name %>,
+
+This message acknowledges receipt of your member emeritus request.
+Please note that you have ten days upon receipt of this request to rescind it per Section 4.3 of the bylaws.
+After that, you will automatically be unsubscribed from the members@apache.org mailing list, and your membership records will be moved to emeritus.
+
```

Comment:
Does it make sense to cancel the subscription?

Reply:
That's supposed to happen in the ten-days-later part of the action. I haven't coded anything for it yet, but it's supposed to unsubscribe the member and move their info to the emeritus section in members.txt and on the website.